src/pages/ReimbursementAccount/BankInfo/BankInfo.tsx

```diff
@@ -98,7 +109,7 @@ function BankInfo({reimbursementAccount, reimbursementAccountDraft, plaidLinkTok
     return (
         <ScreenWrapper testID={BankInfo.displayName}>
             <HeaderWithBackButton
-                shouldShowBackButton={!(setupType === CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID && screenIndex === 0)}
+                shouldShowBackButton
```

Comment:
What is the reason of removing this condition?

Reply:
Because I see no reason to keep it now that we know how to handle the back press in any case (which I've changed the logic as you can see above).